README.md

```
@@ -142,31 +142,123 @@ Parse Dashboard is continuously tested with the most recent releases of Node.js
+| `sslCert` | String | yes | - | `--sslCert` | `PARSE_DASHBOARD_SSL_CERT` | `"/path/cert"` | Path to SSL certificate for HTTPS. |
+| `cookieSessionSecret` | String | yes | Random | `--cookieSessionSecret` | `PARSE_DASHBOARD_COOKIE_SESSION_SECRET` | `"secret"` | Secret for session cookies (for multi-server). |
+| `cookieSessionMaxAge` | Number | yes | Session-only | `--cookieSessionMaxAge` | `PARSE_DASHBOARD_COOKIE_SESSION_MAX_AGE` | `3600` | Session cookie expiration (seconds). |
+| `dev` | Boolean | yes | `false` | `--dev` | - | - | Development mode (**DO NOT use in production**). |
+| `config` | String | yes | - | `--config` | - | `"config.json"` | Path to JSON configuration file. |
+
+#### Helper CLI Commands
+
+| Command | Description |
+|---------|-------------|
+| `--createUser` | Interactive tool to generate secure user passwords and MFA secrets. |
+| `--createMFA` | Interactive tool to generate MFA secrets for existing users. |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)